### PR TITLE
Feature/okxwallet

### PR DESCRIPTION
**Pull request by okxwallet**
### Checklist
- [X] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [X] Check the box that allows repo maintainers to update this PR
- [X] Test locally to make sure this feature/fix works
- [X] Run `yarn check-all` to confirm there are not any associated errors
- [X] Confirm this PR passes Circle CI checks
- [X] Add or update relevant information in the documentation

### Docs Checklist
- [X] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [X] Add/update the appropriate package README (if applicable)
- [X] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [X] Add/update the related package in the `docs/package.json` file (if applicable)

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected)
- [ ] send transaction
- [ ] switch chains
- [ ] sign message
- [ ] sign typed message
- [ ] disconnect

#### Tests with demo app (SDK)
- [X] send transaction
- [X] switch chains
- [X]sign message
- [X] sign typed message
- [X] disconnect

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "docs",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (Adamj1232) [inline docs/package.json:3]:
```suggestion
  "version": "1.0.0",
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "web3-onboard-monorepo",
-  "version": "2.23.1",
+  "version": "2.23.2",
```
Comment (Adamj1232) [inline package.json:3]:
```suggestion
  "version": "2.23.1",
```

File: packages/okxwallet/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+`npm i @web3-onboard/okxwallet`
+
+## Usage
+
+```typescript
+import Onboard from '@web3-onboard/core'
+import okxWalletModule from '@web3-onboard/okxwallet'
+
+// initialize the module with options
+const okxWalletSdk = okxWalletModule({ darkMode: true })
```
Comment (Adamj1232) [inline packages/okxwallet/README.md:14]: Please add available types for okx wallet init options

File: packages/okxwallet/package.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@web3-onboard/okxwallet",
+  "version": "2.0.1",
```
Comment (Adamj1232) [inline packages/okxwallet/package.json:3]:
```suggestion
  "version": "2.0.0-alpha.1",
```

File: packages/okxwallet/README.md (hunk 3 above)
Comment (Adamj1232) [reply packages/okxwallet/README.md:14]: Actually it doesnt look like there are available init options so those should be removed from this

**Comment by Adamj1232** [comment]
@okxwallet We are planning to introduce a new feature that enables the inclusion of a URL as part of the injected module. The intention behind this is to obviate the necessity of a separate module for routing users without installed wallets in the case of injected wallets.  This will help immensely with Blocknative's upkeep and standardize usage for injected wallets without requiring dapp devs to install separate packages for this feature. 
In terms of your PR, we can close this for now and I will update you through github and our telegram channel as we gain clarity on this feature release. I appreciate your cooperation.

Early next week, I will provide updates concerning the method and schedule for accommodating the webpage within the injected module. Thank you!

**Comment by okxwallet** [comment]
> @okxwallet We are planning to introduce a new feature that enables the inclusion of a URL as part of the injected module. The intention behind this is to obviate the necessity of a separate module for routing users without installed wallets in the case of injected wallets. This will help immensely with Blocknative's upkeep and standardize usage for injected wallets without requiring dapp devs to install separate packages for this feature. In terms of your PR, we can close this for now and I will update you through github and our telegram channel as we gain clarity on this feature release. I appreciate your cooperation.
> 
> Early next week, I will provide updates concerning the method and schedule for accommodating the webpage within the injected module. Thank you!

got it

**Comment by Adamj1232** [comment]
@okxwallet we have a PR open to add a feature to easily add this ability without creating/maintaining a new module just for adding the ability to install a wallet that is currently installed. This approach will also be a lighter lift on the dapp developer as they will not need to install any new packages or keep those packages up to date for consistent behavior and only need to add the wallet label to the `displayUnavailable` prop array on injected module initialization. This is the PR - https://github.com/blocknative/web3-onboard/pull/1754
I have added the OKX download URL noted in this PR but feel free to comment or add questions. Thanks!

**Pull request closed without merging**